src/devices/lacrosse_tx22uit.c

```diff
@@ -0,0 +1,137 @@
+/** @file
+    LaCrosse Technology View TX22U-IT temperature, humidity, wind speed/direction
+    and rain sensor.
+
+    Copyright (C) 2020 Caz Yokoyama, caz at caztech dot com
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+/**
+LaCrosse Technology View TX22U-IT temperature, humidity, wind speed/direction
+and rain sensor
+
+Product pages:
+TBW
+
+Specifications:
+TODO: update for TX22U-IT
+- Wind Speed Range: 0 to 178 kmh
+- Degrees of Direction: 360 deg with 16 Cardinal Directions
+- Outdoor Temperature Range: -29 C to 60 C
+- Outdoor Humidity Range: 10 to 99 %RH
+- Update Interval: Every 31 Seconds
+
+http://nikseresht.com/blog/?p=99 tell protocol but my TX22U-IT is different protocol.
+*/
+
+#include "decoder.h"
+
+static int lacrosse_tx22uit_decode(r_device *decoder, bitbuffer_t *bitbuffer)
+{
+    uint8_t const preamble_pattern[] = { 0xaa, 0xaa, 0x2d, 0xd4 };
+
+    data_t *data;
+    uint8_t *b = bitbuffer->bb[0];
+    int offset;
+    static int raw_temp = 0, humidity = -1, raw_speed = -1, direction = -1;
```

Comment:
You should output only the data that appears in a message. Don't use static to hold on to old data.
You can use `DATA_COND` to easily choose what fields to output in data_make.

Reply:
done.